add-image-and-video-dimensions.js

```diff
@@ -0,0 +1,45 @@
+const fs = require('fs');
+
+const glob = require('glob');
+const imageSize = require('image-size');
+const videoSize = require('get-video-dimensions');
+
+(async () => {
+  const files = glob.sync('src/**/*.{html,md}');
+  for (const file of files) {
+    const contents = fs.readFileSync(file, 'utf8').toString();
+    let updatedContents = contents;
+    const images = contents.matchAll(/^\s*<img.*$/gm);
```

Comment:
Looks like copy-paste mistake. I suppose this should check for videos?

Reply:
Ah, I see, you merged both scripts. Then it should check for both images and videos, rather than continuing right away. Otherwise, a page with just videos and no images would currently be skipped altogether.

Reply:
Very good catch. I have changed the logic now.